modules/gapi/perf/common/gapi_render_perf_tests.cpp

```diff
@@ -0,0 +1,1238 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+#include "../perf_precomp.hpp"
+
+#include <iostream>
```

Comment:
I'm not sure that it's really should be here

Reply:
Removed